ray-operator/config/samples/ray-job.tpu-v6e-256-multihost.yaml

```diff
@@ -59,6 +61,20 @@ spec:
                     ephemeral-storage: 20Gi
```

Comment:
One more nit: this example should not include ephemeral-storage limit and use cpu: 24